jsonnet/kube-prometheus/components/prometheus.libsonnet

```
@@ -34,7 +34,15 @@ local defaults = {
     _config: {
       prometheusSelector: 'job="prometheus-' + defaults.name + '",namespace="' + defaults.namespace + '"',
       prometheusName: '{{$labels.namespace}}/{{$labels.pod}}',
-      thanosSelector: 'job="thanos-sidecar"',
+      thanos: {
```

```suggestion
      # TODO: remove `thanosSelector` after 0.10.0 release
      thanosSelector: '',
      thanos: {
```

backwards compatibility layer